resources/stylesheet.css

```diff
@@ -28,6 +28,28 @@ html.darkMode {
 	--tertiaryHover: #27292c;
 }
 
+html.terminalMode {
+--primary: black;
+--secondary: #333;
+--secondaryHover: grey;
+--textPrimary: green;
```

Comment:
I think a color closer to #7af850 would be a bit better. green is a little dark for my preference

Reply:
@gosoccerboy5 see [this](https://github.com/theforumhelpers/theforumhelpers.github.io/pull/243#issuecomment-860894615). if it is still relevant ask @Steve0Greatness

Reply:
@gosoccerboy5 `#32cd32` is what the green is going to look like. Do you still think it should be changed

Reply:
> @gosoccerboy5 `#32cd32` is what the green is going to look like. Do you still think it should be changed

oh wow I didn't know you can preview colours like that